### ISSUE TITLE
1660 ti

### ISSUE THREAD

**Issue by JasonRuan5** [AlexeyAB/darknet#5057]
Hi @AlexeyAB,

I got my laptop with GTX 1660Ti.  I used Yolo in darknet to detect objects in a big image, it takes about 4 minutes, while the other laptop with GTX 1060 takes about a minute.  I searched online and found #3891 (GTX 1060 vs GTX 1660).  I've also changed CUDNN_HALF = 0, it speeds up, but still takes 2.3 minutes.  I wonder what else I can do to improve the speed?   Thanks!

**Comment by AlexeyAB** [comment]
Compile Darknet with GPU=1 CUDNN=1
It should take about ~0.1 sec

**Comment by JasonRuan5** [comment]
Yes, I did compile Darknet with GPU=1 CUDNN=1.  The width = height = 416 in the cfg-file.  We have tested with the same image (40,000 x 40,000) and same detector.
